project/src/views/new-evaluations/newEvaluationsViews.js

```diff
@@ -24,356 +24,98 @@ const CandidateDetailsForm = function() {
     `;
 }
 
-const TechnicalLevelPicker = function() {
+const TechnicalLevelHeader = function(options){
+    const headings = options.headings.map((el) => `<th>${el}</th>`);
```

Comment:
Uuuu! Niiice! Me likey! Can you 'splain what's going on here, though? :)

Reply:
headings will be an array of strings containing all options.headings elements between th tags 💃